src-electron/db/query-impexp.js

```diff
@@ -678,7 +679,9 @@ WHERE
   if (storagePolicy == dbEnums.storagePolicy.attributeAccessInterface) {
     attribute.storageOption = dbEnums.storageOption.external
   }
-
+  if (attributeId == dbEnums.clusterRevision) {
```

Comment:
You are also basing the logic on primary key. Nobody is guaranteeing you that the primary key will ever be the same.
You can't base any logic based on primary keys.